src/qurveros/optspacecurve.py

```diff
@@ -365,4 +361,4 @@ def save_bezier_control_points(self, filename):
         ])
 
         df = pd.DataFrame(points)
-        df.to_csv(filename, index=False, float_format='%.12f')
+        df.to_csv(filename, index=False, float_format='%.12f')
```

Comment:
Using a newline at the end of a file is good Python style! Please leave it in :)